packages/story-editor/src/components/canvas/utils/useSnapping.js

```diff
@@ -71,12 +77,12 @@ function useSnapping({
     [isDragging, designSpaceGuideline, triggerOnboarding]
   );
 
-  if (!canvasContainer || !pageContainer) {
+  if (!canvasContainerBB || !pageContainerBB) {
     return {};
   }
 
-  const canvasRect = canvasContainer.getBoundingClientRect();
-  const pageRect = pageContainer.getBoundingClientRect();
+  const canvasRect = canvasContainerBB;
```

Comment:
Maybe just use `canvasRect` above instead of `canvasContainerBB` and `pageRect` instead of `pageContainerBB`?